package.json

```diff
@@ -24,7 +24,10 @@
     "typecheck": "turbo run typecheck --parallel",
     "clean": "turbo run clean --parallel && yarn clean:root",
     "clean:root": "rm -rf node_modules",
-    "knip": "knip --no-gitignore"
+    "clean:workspaces": "turbo run clean --parallel && yarn",
+    "knip": "knip --no-gitignore",
+    "check-references": "workspaces-to-typescript-project-references --check",
+    "fix-references": "workspaces-to-typescript-project-references && prettier -w \"{apps/*,packages/*/*}/tsconfig.json\""
```

Comment:
Because it causes changes to a JSON file but doesn't format the JSON file in the same way that prettier would, so we prettify it after the change is made.